manifests/f/FireFox2000000/MoonscraperWin32/1.4.3.4/FireFox2000000.MoonscraperWin32.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: FireFox2000000.MoonscraperWin32
+PackageVersion: 1.4.3.4
+PackageLocale: en-US
+Publisher: FireFox2000000
```

```suggestion
Publisher: Alexander (FireFox) Ong
```
This must match Control Panel.